module3/projects/take_home/rubric.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Take-Home Rubric
+layout: page
+---
+
+In order for a candidate to pass this challenge, **_they must score "Leaning Yes Hire" or "Strong Yes Hire" in every category._**
+
+<section class="dropdown">
+### 1. Problem Solving and Task Completion
+#### Strong Yes Hire
+
+The candidate...
+- is able to articulate multiple approaches and describe the pros and cons of each and is able to fully implement the approach, without any real help from the interviewer.
+- recognizes edge cases, draws attention to them unprompted, and solves them.
+- goes above and beyond the MVP in either adding new functionality beyond the base requirements or special consideration through means of testing or accessibility. 
+
+#### Leaning Yes Hire
+The candidate...
+- is able to outline a solution beyond the most basic solution, covering the vast majority of edge cases. 
+- asks questions to clarify the scope of the problem or states their assumptions without being prompted. 
+- is able to modify their solution independently if an edge case is pointed out.
+- builds a solution that meets all MVP expectations provided by the challenge and is easy to use by the end user.
+
+#### Leaning Don’t Hire
+The candidate...
+- is unable to arrive at a solution beyond the most basic approach. The final solution does not account for edge cases. 
+- dives in to a solution without giving the problem much thought. 
```

Comment:
To drive home the importance of planning, what do you think about: 
"dives in to a solution without giving the problem much thought or demonstrating a planning process"

Reply:
oh I now see the whole section for planning below. Nevermind!